scout/server/blueprints/variants/controllers.py

```diff
@@ -1002,7 +1002,7 @@ def generate(header, lines):
     for variant in variant_objs.limit(EXPORTED_VARIANTS_LIMIT):
         variant_line = []
         variant_line.append(str(variant.get("variant_rank", "")))  # index
-        variant_line.append(variant.get("str_repid", ""))  # Repeat locus
+        variant_line.append(variant.get("str_repid", variant.get("str_trid", "")))  # Repeat locus
```

Comment:
Absolutely, good catch! In the web version, we go on to check for any attached gene as well if this is unset. But better already!